### PR TITLE
feat: add aside mdx component

### LINKED ISSUE #532: Aside MDX component

### PR DESCRIPTION
**Pull request by alisonjoseph**
Closes #532 

#### Testing / reviewing

http://localhost:3000/mdx/aside
https://gatsby-theme-carbon.vercel.app/components/Aside

**Note:** The new grid doesn't support offset columns in the middle of a "row", so we will have to update grid mdx slightly for the one page on the Carbon website (inside of data viz) using this component.

### CODE REVIEW

File: services/web-app/components/aside/index.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
```
Comment (francinelucca) [inline services/web-app/components/aside/index.js:2]:
```suggestion
 * Copyright IBM Corp. 2022, 2022
```

File: services/web-app/components/aside/aside.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
```
Comment (francinelucca) [inline services/web-app/components/aside/aside.js:2]:
```suggestion
 * Copyright IBM Corp. 2022, 2022
```

**Review by francinelucca — CHANGES_REQUESTED** [review]

File: services/web-app/components/aside/aside.module.scss [hunk 3]
```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright IBM Corp. 2021, 2022
```
Comment (francinelucca) [inline services/web-app/components/aside/aside.module.scss:2]:
```suggestion
 * Copyright IBM Corp. 2022, 2022
```

**Review by francinelucca — APPROVED** [review]
👯

**Review by andreancardona — APPROVED** [review]

**Pull request merged**